components/SignupForm/SignupForm.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from "react";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { Layout } from "../Layout";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import styled from "@emotion/styled";
+import {
+  AuthFormType,
+  loginForm,
+  signupForm,
+  forgotPasswordForm,
+  updatePasswordForm,
+  updateEmailForm
+} from "../AuthForm/constants";
+import { useAuth } from "../../config/auth";
+
+const FieldContainer = styled.div`
```

Comment:
Not a very big deal, but we've now duplicated FieldContainer and its styles in AuthForm and in SignupForm - is it worthwhile to extract this? (not sure, in some cases copy & paste is the better solution as you *might* want to apply different styles for the different forms at a later stage)